tools/bazar/presentation/javascripts/bazar-list-dynamic.js

```diff
@@ -0,0 +1,205 @@
+                  if (Array.isArray(fieldValue)) fieldValue = fieldValue.join(' ')
+                  fieldValue = this.removeDiatrics(fieldValue)
+                  if (fieldValue && fieldValue.includes(word)) {
+                    entry.searchScore += field == 'bf_titre' ? 2 * word.length : word.length
+                  }
+                })
+              })
+              return entry.searchScore > 0
+            }
+            return true
+          })
+          if (needSearch) result = result.sort((a, b) => (a.searchScore > b.searchScore) ? -1 : 1)
+        }
+        this.searchedEntries = result
+        this.filterEntries()
+      },
+      filterEntries() {
+        // Handles filters
+        let result = this.searchedEntries
+        for(const filterId in this.computedFilters) {
+          result = result.filter(entry => {
+            if (!entry[filterId]) return false
+            return entry[filterId].split(',').some(value => {
+              return this.computedFilters[filterId].includes(value)
+            })
+          })
+        }
+        this.filteredEntries = result
+        this.paginateEntries()
+      },
+      paginateEntries() {
+        let result = this.filteredEntries
+        if (this.pagination > 0) {
+          let start = this.pagination * this.currentPage
+          result = result.slice(start, start + this.pagination)
+        }
+        this.paginatedEntries = result
+        this.formatEntries()
+      },
+      formatEntries() {
+        this.paginatedEntries.forEach(entry => {
+          entry.color = this.colorIconValueFor(entry, this.params.colorfield, this.params.color)
+          entry.icon  = this.colorIconValueFor(entry, this.params.iconfield, this.params.icon)
+        })
+        this.entriesToDisplay = this.paginatedEntries
+      },
+      calculateFiltersCount() {
+        for(let fieldName in this.filters) {
+          for (let option of this.filters[fieldName].list) {
+            option.nb = this.searchedEntries.filter(entry => {
+              let entryValues = entry[fieldName]
+              if (!entryValues) return
+              entryValues = entryValues.split(',')
+              return entryValues.some(value => value == option.value)
+            }).length
+          }
+        }
+      },
+      resetFilters() {
+        for(let filterId in this.filters) {
+          this.filters[filterId].list.forEach(option => option.checked = false)
+        }
+        this.search = ''
+      },
+      field(entry, field, fallbackField) {
+        let mappedField = this.params.displayfields[field]
+        return mappedField ? entry[mappedField] : entry[fallbackField]
+      },
+      getEntryRender(entry) {
+        if (entry.html_render) return
+        let fieldsToExclude = this.params.displayfields ? Object.values(this.params.displayfields) : []
+        $.getJSON(`?api/entry/${entry.id_fiche}/view&excludeFields=${fieldsToExclude}`, function(data) {
```

Comment:
```suggestion
        $.getJSON(`?api/entries/html/${entry.id_fiche}&fields=html_output&excludeFields=${fieldsToExclude}`, function(data) {
```

je ne suis pas partisan pour la création d'une nouvelle route api (ça rend alors plus difficile la maintenance de toutes les routes et la documentation)